Parsers/ASimNetworkSession/ProductParsers/ASimNetworkSessionPaloAltoPanOS.yaml

```diff
@@ -0,0 +1,118 @@
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let NWParser=(disabled:bool=false){
+  CommonSecurityLog | where not(disabled)
+  | where DeviceVendor=="Palo Alto Networks" and (Activity=="TRAFFIC")
+  | parse AdditionalExtensions with "PanOSDstPackets="DstPackets:long";PanOSsrc_packets="SrcPackets:long";start="EventStartTime:datetime";reason="*
+  | project-rename 
+        EventVendor=DeviceVendor // Not documented
+      , EventProduct=DeviceProduct // Not Documented
+      , EventProductVersion=DeviceVersion // Not Documented
+      , DvcHostname=DeviceName   
+      , EventResourceId=_ResourceId  // Not Documented 
+      , NetworkApplicationProtocol=ApplicationProtocol
+      , SrcZone=DeviceCustomString4 
+      , DstZone=DeviceCustomString5 
+      , NetworkRuleName=DeviceCustomString1
+      , NetworkProtocol=Protocol
+      , SrcBytes=SentBytes
+      , DstBytes=ReceivedBytes 
+      , SrcUsername=SourceUserID 
+      , DstUsername=DestinationUserID 
+      , EventSeverity=LogSeverity // not documented
+      , NetworkPackets=DeviceCustomNumber2 
+      , SrcNatIpAddr=SourceTranslatedAddress
+      , DstNatIpAddr=DestinationTranslatedAddress
+      , NetworkConnectionHistory=FlexString1 // Flags
+      // , UrlCategory=DeviceCustomString2 -- websessions
+      // , LogProfile=DeviceCustomString6  
+      //, VirtualSystem=DeviceCustomString3
+    // Proxy
+    // , UrlOriginal=RequestURL
+    // , HttpUserAgentOriginal=RequestClientApplication
+    // , HttpContentType=RequestContext
+    // , HttpRequestMethod=RequestMethod
+  | extend
+      NetworkBytes=tolong(FlexNumber1)
+      , SrcUsernameType="UPN"
+      , DstUsernameType="UPN"
+      , TimeGenerated  
+      , EventType="NetworkSession" // it is not clear from doc when NWConnection and when NWSession
+      , EventCount=toint(1)
+      , EventResult=case(DeviceAction=="allow","Success","Failure")
+      , NetworkSessionId=tostring(DeviceCustomNumber1)
+      , NetworkDuration=DeviceCustomNumber3
+      , EventSchemaVersion="0.2.0"
+      , EventSchema="imNetworkSession"
+      , DvcAction=case(DeviceAction=="reset client", "Reset Source",  DeviceAction=="reset server", "Reset Destination",  DeviceAction=="reset both", "Reset",  DeviceAction)
+  | project-rename
+        // DvcMacAddr=DeviceMacAddress 
+      , DstHostname=DestinationHostName
+      , DstMacAddr=DestinationMACAddress
+    , SrcMacAddr=SourceMACAddress
+      , DstIpAddr=DestinationIP
+      , DstPortNumber=DestinationPort
+      , DstNatPortNumber=DestinationTranslatedPort
+    , SrcPortNumber=SourcePort
+      , SrcIpAddr=SourceIP
+    , DstUserName=DestinationUserName
+    , SrcNatPortNumber=SourceTranslatedPort
+    , DvcOutboundInterface=DeviceOutboundInterface
+    , DvcInboundInterface=DeviceInboundInterface
+    // Trivial renames to mitigate Autocomplete
+    /// , NetworkDirection: no such data 
+    //  , EventEndTime=EndTime
+    // , EventMessage=Message
```

Comment:
I think this should not be commented out.

Reply:
ok, but eventmessage is not documented. (I know it used to be..)